### PR TITLE
Universal wallet

### PR DESCRIPTION
**Pull request by moskalyk**

### CODE REVIEW

File: src/boilerplates/boilerplates.ts [hunk 1]
```diff
@@ -116,6 +117,21 @@ export function makeCommandBoilerplates(program: Command) {
             createEmbeddedWalletVerifySession(program, options);
         });
 
+    comm
+        .command("create-universal-wallet-react")
```
Comment (RichardIrala) [inline src/boilerplates/boilerplates.ts]: If this is a boilerplate and also a starting point, I would call it create-universal-wallet-react-starter.

**Comment by RichardIrala** [comment]
* You should add the new command to readme.md
* The command works well

File: src/boilerplates/boilerplates.ts (hunk 1 above)
Comment (taylanpince) [reply src/boilerplates/boilerplates.ts]: Agreed – @moskalyk Also add the new boilerplate to the README please.

**Review by taylanpince — CHANGES_REQUESTED** [review]
Looks good, minor naming change only

**Review by RichardIrala — APPROVED** [review]

**Review by taylanpince — APPROVED** [review]

**Pull request merged**